src/services/drive/database/driveLocalDB.ts

```diff
@@ -137,9 +138,12 @@ class DriveLocalDB {
         folderId: row.parent_id as number,
         folder_id: row.parent_id as number,
         name: row.name,
+        plain_name: row.plain_name,
```

Comment:
please use camelCase always if is not a limitation (SQLite etc)

Reply:
Okay! 
Here I have used it this way because it is the SDK typing, because from what I saw when getting the columns from the Postgres tables you get the field with the name as it is in the column and there is nothing to do a parsing, that's why they are in snake case. 
If I'm wrong about this I can check it and modify it!